File: src/main/java/com/example/semesterwork/review/ReviewResponse.java
package com.example.semesterwork.review;

import com.example.semesterwork.user.dto.UserDto;
import com.example.semesterwork.user.model.UserModel;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Builder
@AllArgsConstructor
@NoArgsConstructor
@Data
public class ReviewResponse {
    private Float rating;
    private String text;
    private UserDto user;
}

File: frontend/app/src/main/java/com/example/travels/data/review/ReviewResponse.kt
package com.example.travels.data.review

import com.example.travels.data.user.remote.response.UserResponse
import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class ReviewResponse(
    @SerialName("rating") val rating: Float,
    @SerialName("text") val text: String,
    @SerialName("user") val user: UserResponse,
)
